public/app/features/alerting/unified/components/rule-editor/NotificationsStep.tsx

```diff
@@ -25,6 +26,12 @@ enum RoutingOptions {
   ContactPoint = 'contact point',
 }
 
+function useHasInternalAlertmanagerEnabled() {
+  const { currentData: amChoiceStatus } =
+    alertmanagerApi.endpoints.getGrafanaAlertingConfigurationStatus.useQuery(undefined);
```

Comment:
Could we destructure the `useQuery` from the API instead please? (just following on from the Slack discussion the other day)

```
const { useGetGrafanaAlertingConfigurationStatusQuery } = alertmanagerApi;
...
```

Reply:
truee, I forgot it!